app/Resources/views/pagelayout.html.twig

```diff
@@ -43,17 +43,19 @@
         {% endblock %}
 
             {% block header %}
-                {% include 'pagelayout_header.html.twig' %}
+                {% include 'layout/header.html.twig' %}
             {% endblock %}
 
             {% block content %}
             {% endblock %}
 
             {% block footer %}
                 {{ render(controller('AppBundle:Layout:showPart', {
-                      'template': 'pagelayout_footer.html.twig'
+                      'template': 'layout/footer.html.twig'
                 })) }}
             {% endblock %}
+            {% include 'layout/scripts.html.twig' %}
+            {% block javascripts %}{% endblock %}
```

Comment:
@clash82  not yet I set this up like on demo, right now all js are on head_scripts